### ISSUE TITLE
Save-Sync not using the specified path

### ISSUE THREAD

**Issue by Mereck13579** [Heroic-Games-Launcher/HeroicGamesLauncher#2334]
### Describe the bug

On Death Stranding (unsure if it affects other games as I only played this one so far), I have the game both on my SteamDeck and on my Windows PC. On Windows specifically, when I try to download the save files, the last character from the folder name gets deleted, here are some screenshot of the setting + windows explorer:
Sync-Save setting:
![Screenshot 2023-01-12 134133](https://user-images.githubusercontent.com/7541087/212069202-f03b6d7e-a5d3-4b28-9b2c-c1bd15b615f2.png)
File Explorer of the folder prior to syncing:
![heroicbug2](https://user-images.githubusercontent.com/7541087/212069352-e01408bd-9a10-4352-ba83-22d8026300ce.png)
File Explorer of the folder after syncing:
![heroicbug1](https://user-images.githubusercontent.com/7541087/212069378-83ceaa93-516b-47a9-bd99-80193ece9c7e.png)

(removed my name from the logs output)
Added both regular and syncing logs (syncing logs clearly states a wrong path despite being set correctly)

### Add logs

```text
System Info:
Heroic Version: 2.5.2 Yamato
Legendary Version:  0.20.31 Dark Energy (hotfix #5)
GOGdl Version: 0.5
OS: Microsoft Windows 10 Pro KERNEL: 10.0.19044 ARCH: x64
CPU: AMD Ryzen 7 5700X 8-Core Processor @3.4 
RAM: Total: 31.89 GiB Available: 20.24 GiB
GRAPHICS: GPU0: NVIDIA GeForce RTX 3070 VRAM: 8192MB DRIVER: 516.94 


Game Settings: {
	"autoInstallDxvk": true,
	"autoInstallVkd3d": true,
	"preferSystemLibs": false,
	"nvidiaPrime": false,
	"enviromentOptions": [],
	"wrapperOptions": [],
	"savesPath": "{appdata}/../locallow/KojimaProductions/DeathStranding/{EpicID}/",
	"showFps": false,
	"useGameMode": false,
	"language": ""
}

Game launched at: Thu Jan 12 2023 11:05:55 GMT+0100 (heure normale d’Europe centrale)

Legendary's config from config.ini (before Heroic's settings):
{
  "game_parameters": [],
  "game_executable": "DeathStranding.exe",
  "game_directory": "H:\\Games\\DeathStranding",
  "launch_command": [],
  "working_directory": "H:\\Games\\DeathStranding",
  "user_parameters": [],
  "environment": {},
  "pre_launch_command": "",
  "pre_launch_wait": false
}

Launch Command: C:\Users\XXX\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary launch Boga --language en 

Game Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[Core] INFO: Getting ownership token.
[cli] INFO: Launching Boga...

Legendary update available!
- New version: 0.20.32 - "Dark Energy (hotfix #6)"
- Release summary:
 [!] Fixes installing games that specify invalid install directories (e.g. Kerbal Space Program)
 [*] More gracefully handles account errors
See full changelog for additional details.
- Release URL: https://legendary.gl/release/0.20.32

- Download URL: https://legendary.gl/release/download/0.20.32/legendary.exe

___________________________________________________________________________________________________

Sync-Saves
Progress:
[Core] INFO: Trying to re-use existing login session...

[cli] INFO: Got 1 remote save game(s)

[cli] INFO: Checking "Death Stranding" (Boga)

[cli] INFO: Overriding save path with "C:\Users\XXX\AppData\LocalLow\KojimaProductions\DeathStranding\b7f376d8f6a54f95800294de7d6e6da"...

[cli] INFO: Cloud save for "Death Stranding" is newer:

[cli] INFO: - Cloud save date: 2023-01-11 12:08:10

[cli] INFO: - Local save date: N/A

[cli] INFO: Downloading remote savegame...

[Core] INFO: Deleting old save files...

[Core] INFO: Downloading "Boga/manifests/2023.01.11-12.08.10.manifest"...

[Core] INFO: Successfully completed savegame download.



Legendary update available!

- New version: 0.20.32 - "Dark Energy (hotfix #6)"

- Release summary:

 [!] Fixes installing games that specify invalid install directories (e.g. Kerbal Space Program)

 [*] More gracefully handles account errors

See full changelog for additional details.

- Release URL: https://legendary.gl/release/0.20.32



- Download URL: https://legendary.gl/release/download/0.20.32/legendary.exe
```

### Steps to reproduce

1. Go to Game Settings -> Save Sync
2. Specify a path
3. Manually download saves
4. Download complete successfully but saves are on the wrong location

### Expected behavior

Saves should be placed in the specified directory

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Ubuntu"]: Windows 10 PRO Build 19044

**Comment by Mereck13579** [comment]
Found a workaround: adding a trailing \ at the end of the path fixes it, I'm suspecting something eats up the last character of the path, whether it's a \ or not.

**Comment by flavioislima** [comment]
Fixed by #2319
